wire-ios-mono.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -46,6 +46,15 @@
           "version": "1.23.0"
         }
       },
+      {
+        "package": "libPhoneNumber",
+        "repositoryURL": "https://github.com/wireapp/libPhoneNumber-iOS.git",
+        "state": {
+          "branch": null,
+          "revision": "a6f5cb0a2a264e8396da65b0fb27ecf0df8a975e",
+          "version": "1.1.0"
+        }
+      },
```

Comment:
we should update it and not use the fork -> not sure we should move it from carthage if we're aiming to remove it soon